tests/e2e/helpers.go

```diff
@@ -284,34 +284,43 @@ func executeTests(t *testing.T, tests ...testExecution) {
 	// defer deleting the profiles or else the test namespace get stuck in Terminating
 	defer f.Client.Delete(goctx.TODO(), ocp4Pb)
 
-	t.Run("Parallel tests", func(t *testing.T) {
-		for _, test := range tests {
-			// Don't loose test reference
-			test := test
-			if test.IsParallel {
-				t.Run(test.Name, func(tt *testing.T) {
-					tt.Parallel()
-					if err := test.TestFn(tt, f, ctx, mcTctx, ns); err != nil {
-						tt.Error(err)
-					}
-				})
+	testtype := ctx.GetTestType()
+	if testtype == framework.TestTypeAll || testtype == framework.TestTypeParallel {
+		t.Run("Parallel tests", func(t *testing.T) {
+			for _, test := range tests {
+				// Don't loose test reference
```

Comment:
loose or lose?

Reply:
English is hard. I should just stick to Spanish 😛